languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -10061,4 +10061,1404 @@ Ben Ali
+Marie von Ebner-Eschenbach
+Svetlana Aleksievich
+Vasil Bykaŭ
+Adolphe Quetelet
+Amélie Nothomb
+Charles De Coster
+Emile Verhaeren
+Fernand Khnopff
+François-Joseph Fétis
+Georges Simenon
+Guido Gezelle
+Hergé
+Hugo Claus
+Maurice Maeterlinck
+Stijn Streuvels
+Henri Michaux
+Pierre Alechinsky
+Mikhail Gurevich
+Christoph Martin Wieland
+Johann Christoph Gottsched
+Caetano Veloso
+Jorge Amado
+Machado de Assis
+Oscar Niemeyer
+Paulo Freire
+Lafcadio Hearn
+Sheridan Le Fanu
+Muhammad Iqbal
+Rabindranath Tagore
+Ivan Vazov
+Alice Munro
+Lucy Maud Montgomery
+Marshall McLuhan
+Maya Angelou
+Michael Ignatieff
+Michael Ondaatje
+William Osler
+Alberto Manguel
+Josef Škvorecký
+Elizabeth Bishop
+Ernest Thompson Seton
+John Kenneth Galbraith
+James K. Galbraith
+Natalie Zemon Davis
+Simon Newcomb
+William Gibson
+Cory Doctorow
+Pablo Neruda
+Jan Kochanowski
+Pavel Kohout
+Bohumil Hrabal
+Petr Bakalář
+Jaroslav Seifert
+Tomáš Garrigue Masaryk
+Jaroslav Hašek
+Adam Oehlenschläger
+Jens Peter Jacobsen
+Theodor Mommsen
+Christian Friedrich Hebbel
+Giovanni Pico della Mirandola
+Theodor Storm
+Anna Maria van Schurman
+Constantijn Huygens
+Gerardus Vossius
+Hugo Grotius
+Romeyn de Hooghe
+Samuel van Hoogstraten
+Karel van Mander the Elder
+Boutros Boutros-Ghali
+Naguib Mahfouz
+Taha Hussein
+Christian Fürchtegott Gellert
+Gotthold Ephraim Lessing
+Novalis
+Joseph Addison
+Roger Williams
+William Jones
+Daniel King
+Neil McDonald
+Philip Sidney
+David S. Goodman
+Alia Dannenberg
+Mika Waltari
+Tove Jansson
+Giuseppe Mazzini
+Abraham Bosse
+Adrien Leroy
+Agrippa d'Aubigné
+Alain
+Alain-René Lesage
+Albert Gleizes
+Alexandra David-Néel
+Alexandre Dumas
+Alexis de Tocqueville
+Alfred de Musset
+Alfred de Vigny
+Alphonse Daudet
+Alphonse de Lamartine
+Alphonse-Marie-Adolphe de Neuville
+Anatole France
+Anaïs Nin
+André Chénier
+André Gide
+André Maurois
+Anna de Noailles
+Anne-Louis Girodet de Roussy-Trioson
+Antoine François Prévost
+Aristide Briand
+Arsène Arnaud Claretie
+Arthur de Gobineau
+Auguste Comte Xavier
+Auguste Villiers de l'Isle-Adam
+Bernard Le Bovier de Fontenelle
+Bruno Latour
+Camille Flammarion
+Catulle Mendès
+Charles Augustin Sainte-Beuve
+Charles Baudelaire
+Charles Forbes René de Montalembert
+Charles Fourier
+Charles Marie René Leconte de Lisle
+Charles Maurras
+Charles Nodier
+Charles Péguy
+Charles Richet
+Charles-Maurice de Talleyrand-Périgord
+Charles-Nicolas Cochin
+Christine de Pizan
+Chrétien de Troyes
+Clara Morgane
+Claude Adrien Helvétius
+Claude Cahun
+Claude Henri de Rouvroy
+Claude Joseph Rouget de Lisle
+Dominique Vivant
+Dora Maar
+Edmond About
+Edmond Rostand
+Edmond de Goncourt
+Emmanuel Joseph Sieyès
+Emmanuel Levinas
+Ernest Renan
+Eugène Fromentin
+Eugène Labiche
+Eugène Scribe
+Eugène Sue
+Ferdinand de Lesseps
+Francis Jammes
+Francis Picabia
+Francis de Sales
+Frantz Fanon
+François Arago
+François Coppée
+François Guizot
+François Mauriac
+François de La Rochefoucauld
+François-Auguste-René de Chateaubriand
```

Comment:
He's much more commonly known as `François-René de Chateaubriand`, no? @LucieSteib ?

Reply:
Yes, definitely

Reply:
Also 'Chateaubriand' is already present on the list, and I'd be surprised if it wasn't already supported by most languages. Either way, this is just because I think that `Auguste` there might be a surprise to a lot of people (it was to me).